src/pages/Logs/Logs.tsx

```diff
@@ -6,6 +8,11 @@ const Logs = (): JSX.Element => (
     <Header>
       <b>Audit logs</b>
     </Header>
+    <div className="l-content logs">
+      <FadeIn isActive={true}>
+        <AuditLogsTable showModel />
```

Comment:
To make only the table scroll this needs to be wrapped in `<div className="u-overflow--auto">...</div>`.